app/scripts/services/validateRegistrant.js

```diff
@@ -8,8 +8,9 @@ angular.module('confRegistrationWebApp')
       var ruleOperand = block.content && block.content.ruleoperand ? block.content.ruleoperand : 'AND';
       var validRuleCount = 0;
 
-      for (var i = 0; i < block.rules.length; i++) {
-        var rule = block.rules[i];
+      Rollbar && Rollbar.info('Block rules value in blockVisibleRuleCheck' + JSON.stringify(block.rules));
```

Comment:
@OzzieOrca 
Should we limit the log to Rollbar only if `!block.rules` (that is, if `block.rules` is not defined)?